src/ImageSharp/Formats/Tiff/PhotometricInterpretation/BlackIsZero8TiffColor.cs

```diff
@@ -24,13 +25,16 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
 
             int offset = 0;
 
+            var rgba = default(Rgba32);
```

Comment:
You can use the `L8` data type directly instead or rgba. Its ment for gray image data

Reply:
👌

Reply:
Done 798603f46cd92c21c4cf0aa74a2e2082b83d7ae7